### PR TITLE
Added Python demo for bert_question_answerer

### PR DESCRIPTION
**Pull request by VihangaAW**
Added a Python wrapper for BertQuestionAnswerer

### CODE REVIEW

File: tensorflow_lite_support/examples/task/text/desktop/python/BUILD [hunk 1]
```diff
@@ -34,3 +34,23 @@ py_binary(
...
+# includes territory belonging to nine nations."
+py_binary(
+    name = "bert_question_answerer_demo",
+    srcs = ["bert_question_answerer_demo.py"],
+    data = ["//tensorflow_lite_support/examples/task/text/desktop:bert_question_answerer_demo"],
+    deps = [
+        "@absl_py//absl:app",
+        "@absl_py//absl/flags",
+    ],
+)
```
Comment (lu-wang-g) [inline tensorflow_lite_support/examples/task/text/desktop/python/BUILD:56]: nit: Please add a new line to the end of the file. This is a coding convention at Google.

**Comment by lu-wang-g** [comment]
Just one minor comment. Thanks for the contribution!

File: tensorflow_lite_support/examples/task/text/desktop/python/BUILD (hunk 1 above)
Comment (VihangaAW) [reply tensorflow_lite_support/examples/task/text/desktop/python/BUILD]: Hi @lu-wang-g! I have updated the pull request.

**Review by lu-wang-g — APPROVED** [review]
Thanks for the updates. I'll approve this for internal review.

**Pull request merged**